docs/pages/eas-insights/introduction.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: EAS Insights
+sidebar_title: Introduction
+hideTOC: true
+description: An introduction to EAS Insights which is a preview service for projects using the expo-insights library.
+---
+
+**EAS Insights** is a service that offers a view into a project's performance,
+usage, and reach. We are currently offering a preview of Insights that is
+available to all developers, and we will continue to roll out new features and
+functionality based on user feedback and suggestions.
+
+EAS Insights makes it easy to see the state of your app, providing information
+about usage across platforms, app store versions, and timeframes.
+
+### Integration with EAS Update
+
+For those already using [EAS Update](/eas-update/introduction.mdx), we're able to provide certain
+high-level usage insights without any additional client-side changes. This is
+possible because by aggregating data from client requests to check for an update into a limited Insights view that shows usage over time, as well
+as usage broken down by platform.
+
+### Using the `expo-insights` module
+
+Developers can add the `expo-insights` module to their project and gain more
+precise usage metrics (than what can be provided by just aggregating update requests), as well as additional breakdowns by app store version.
+Currently, the module is limited to sending client events only
+pertaining to cold starts of the app, but in the future we will expand
+`expo-insights` to offer new types of events and payloads, to support more
+advanced functionality.
```

Comment:
```suggestion
Currently, the module is limited to sending client events only
of cold starts of the app. However, in the future, we will expand
`expo-insights` to offer new types of events and payloads to support more
advanced functionality.
```

Reply:
Not going to change this one, IMO the original language is preferable